main/src/test/java/org/mobilitydata/gtfsvalidator/cli/ArgumentsTest.java

```diff
@@ -122,4 +124,41 @@ public void numThreadsShouldHaveDefaultValueIfNotProvided() {
     assertThat(underTest.getCountryCode()).matches("ca");
     assertThat(underTest.getNumThreads()).isEqualTo(1);
   }
+
```

Comment:
I am not sure that those tests add value because a single option without params is trivial. So I suggest to revert this file.